### ISSUE TITLE
Please add "started downloading..." and "done downloading" bot messages for isolate downloads

### ISSUE THREAD

**Issue by rmistry** [luci/luci-py#266]
For Skia when a new bot is brought online for the first time it spends a lot of time downloading things and populating its cache. This time can sometimes be more than the I/O timeout, in which case the bot then times out due to no output:
https://chromium-swarm.appspot.com/user/task/2fb117cc1aad5a10

It would be very useful to show some indication of what a bot is doing during this time. This would, I imagine, be useful for all swarming users. Maybe something like:
"""
Downloading isolates...
Done downloading!
"""

Optionally we could print out "Downloading isolates..." every X minutes or if you know how many are left to be downloaded "xyz isolates remaining...".

**Comment by mithro** [comment]
Printing out the number of files remaining to be downloaded (and maybe how many have been downloaded so far) every minute seems a pretty good idea.

**Comment by maruel** [comment]
The main reason I had avoided this is that I didn't want to "corrupt" the Swarming task stdout. That said, I agree *something* is needed. I'm open to all ideas. In the meantime is it possible for you to increase the I/O timeout for your tasks?

**Comment by rmistry** [comment]
Yes, we had increased the IO timeout as a workaround. Showing an indication of what the bot is doing during this time would be very helpful.
